src/SourceBuild/tarball/content/build.sh

```diff
@@ -156,11 +156,12 @@ export DOTNET_CLI_TELEMETRY_OPTOUT=1
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 export NUGET_PACKAGES=$restoredPackagesDir/
 
+LogDateStamp=$(date +"%m%d%H%M%S")
```

Comment:
These changes cleanup some log files that were getting created at the tarball's root.